gradle-scripts/extensions.gradle

```diff
@@ -0,0 +1,12 @@
+ext {
+    mockitoVersion = '2.12.0'
+    slf4jSimpleVersion = '1.7.25'
+    jUnitVersion = '4.12'
+    assertjVersion = '3.6.1'
+    checkstyleVersion = '7.7'
+    pmdVersion = '5.6.1'
+    jacocoVersion = '0.7.9'
+    findbugsVersion = '3.0.1'
+    commercetoolsJvmSdkVersion='1.26.0'
+    systemRulesVersion='1.16.0'
```

Comment:
there is a newer one: http://search.maven.org/#artifactdetails%7Ccom.github.stefanbirkner%7Csystem-rules%7C1.17.0%7Cjar

Reply:
e955513